packages/calcite-components/src/components/select/select.scss

```diff
@@ -3,104 +3,112 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-select-background-color: The background color of the component.
+ * @prop --calcite-select-border-color: The border color of the component.
  * @prop --calcite-select-font-size: The font size of `calcite-option`s in the component.
- * @prop --calcite-select-spacing: The padding around the selected option text.
+ * @prop --calcite-select-space-inline: The padding around the selected option text.
+ * @prop --calcite-select-text-color: The text color of the component.
  */
 
 :host {
+  --calcite-select-background-color: var(--calcite-color-foreground-1);
+  --calcite-select-border-color: var(--calcite-color-border-input);
+  --calcite-select-text-color: var(--calcite-color-text-2);
+
+  --calcite-internal-select-icon-space-inline: var(--calcite-color-foreground-1);
+  --calcite-internal-select-border-color: var(--calcite-select-border-color);
```

Comment:
why is there an internal var and component token of the same name here?

Reply:
My bad, I shouldn't have added the internal one. Will update.